### PR TITLE
Update must gather image workflow

### PR DESCRIPTION
**Pull request by Vincent056**
Let's fix the action for must gather, and add multi arch build to it.

### CODE REVIEW

**Review by rhmdnd — COMMENTED** [review]
/lgtm

Only minor suggestion would be to isolate the architecture enablement changes from the other changes, resulting in a series of orthogonal patches (makes reviewing, reverting, and historical tracing easier).

File: .github/workflows/must-gather-latest.yml [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
     uses: metal-toolbox/container-push/.github/workflows/container-push.yml@main
     with:
       name: must-gather-ocp
       registry_org: complianceascode
       tag: latest
       dockerfile_path: images/must-gather/Dockerfile
-      vendor: 'Compliance Operator Authors'
\ No newline at end of file
+      vendor: "Compliance Operator Authors"
+      platforms: "linux/amd64,linux/ppc64le,linux/s390x"
```
Comment (prb112) [inline .github/workflows/must-gather-latest.yml:20]: looks good from my perspective

**Comment by prb112** [comment]
Excellent

**Pull request merged**